### PR TITLE
34 missing the id name description properties

### PR DESCRIPTION
**Pull request by liunelson**
Minor PR. Added the "id", "name", "description" properties that was missing in "transitions" and "parameters" of the Petri net schema and examples.

### CODE REVIEW

File: petrinet/examples/sir.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
       {
         "id": "S",
         "name": "Susceptible",
+        "description": "Number of individuals relative to the total population that are 'susceptible' to a disease infection",
```
Comment (bgyori) [inline petrinet/examples/sir.json:11]: In #17 I added units that assume absolute counts for this example model. Could you remove the "relative to the total population" part of the description?

File: petrinet/examples/sir.json [hunk 2]
```diff
@@ -57,7 +61,8 @@
           "R"
         ],
         "properties": {
-          "name": "Recovery"
+          "name": "Recovery",
+          "description": "Recovery interaction of a infected individual"
```
Comment (bgyori) [inline petrinet/examples/sir.json:65]: Instead of "interaction", we can call this "process".

**Review by bgyori — CHANGES_REQUESTED** [review]

File: petrinet/examples/sir.json (hunk 1 above)
Comment (liunelson) [reply petrinet/examples/sir.json]: Done, I also removed it for `typed_sir.json` in anticipation!

File: petrinet/examples/sir.json (hunk 2 above)
Comment (liunelson) [reply petrinet/examples/sir.json]: Done, I also removed it for `typed_sir.json` in anticipation!

**Comment by liunelson** [comment]
@bgyori I think this is ready to be merged?

**Review by bgyori — APPROVED** [review]

**Pull request merged**